src/helpers/getShowUtils.ts

```diff
@@ -16,24 +17,34 @@ const getShowsByName = async (name: string): Promise<ShowData[] | null> => {
     );
     if (!response.ok) {
         LOG.error('Fetch request failed with a status of ' + response.status);
+        return null;
     }
     const data = (await response.json()) as ShowResults;
     if (!data.results) {
         LOG.error('No results found');
         return null;
     }
     return data.results.map((show) => {
-        return {
+        const obj = {
```

Comment:
I was playing around with this for a bit. Would it be better to do
```
const obj = {
            id: show.id,
            poster_path: show.poster_path,
            title: show.media_type === 'movie' ? (show as MovieData).title : (show as TvData).name,
            release_date: show.media_type === 'movie' ? (show as MovieData).release_date : (show as TvData).first_air_date,
            vote_average: show.vote_average,
            vote_count: show.vote_count,
            overview: show.overview,
            media_type: show.media_type,
            genre_ids: show.genre_ids,
        };
```

Reply:
Yeah that is a bit cleaner, updated.